http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/sam/SamService.scala

```diff
@@ -13,45 +19,45 @@ trait SamService[F[_]] {
 
   /**
    * Gets a user's pet GCP service account, using the user's token.
-   * @param userInfo the user info containing an access token
+   * @param bearerToken the user's access token
```

Comment:
I decided to have these methods take a token string instead of a `UserInfo` object. It was too tempting to use the `userInfo.email` field for logging/etc, but that email should not be relied upon. We should call `SamService.getUserEmail` when we need to resolve the email from a token.